### PR TITLE
changed color in health bar

### PR DESCRIPTION
**Pull request by JPM709**

### CODE REVIEW

File: main/canvas.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ function healthBar(healthRemaining) {
     c.stroke();
 
     //inside
-    c.fillStyle = 'lightcoral'
+    c.fillStyle = 'red'
```
Comment (connected-soft) [inline main/canvas.js:157]: maybe we don't want it to be red?

File: main/canvas.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ function oneCent(x) {
 
 //produce a rectangle that follows mouse and displays information
 function toolTip(x, y, long, tall){
-    c.fillStyle = 'white';
+    c.fillStyle = 'lightblue';
```
Comment (connected-soft) [inline main/canvas.js:169]: LGTM!!

**Review by connected-soft — APPROVED** [review]

File: main/canvas.js (hunk 1 above)
Comment (JPM709) [reply main/canvas.js:157]: I want Red!

**Comment by JPM709** [comment]
finished with branch

**Pull request closed without merging**